invoice/invoice.py

```diff
@@ -45,3 +47,36 @@ def get_sorted_key(self, name):
                 key.append(purchase)
 
         return key
+
+
+class InvoiceReport(HTMLReport):
+    __name__ = 'account.invoice'
+
+    @classmethod
+    def __setup__(cls):
+        super(InvoiceReport, cls).__setup__()
+        cls.__rpc__['execute'] = RPC(False)
+
+    @classmethod
+    def execute(cls, ids, data):
+        pool = Pool()
+        Invoice = pool.get('account.invoice')
+
+        result = super(InvoiceReport, cls).execute(ids, data)
+
+        if len(ids) == 1:
+            # Re-instantiate because records are TranslateModel
```

Comment:
I think this comment does not apply here.

Reply:
copy from https://github.com/tryton/account_invoice/blob/develop/invoice.py#L2666